packages/cli-repl/src/arg-mapper.ts

```diff
@@ -19,23 +17,13 @@ const MAPPINGS = {
   tlsAllowInvalidCertificates: 'tlsAllowInvalidCertificates',
   tlsAllowInvalidHostnames: 'tlsAllowInvalidHostnames',
   tlsCAFile: 'tlsCAFile',
-  tlsCRLFile: { opt: 'sslCRL', val: readCrlFileContent },
+  tlsCRLFile: 'sslCRL',
```

Comment:
They fixed it :) If there’s an issue associated with this, I can’t find it (/cc @rose-m ?)

Reply:
No there wasn't an issue with that - `sslCRL` was officially documented by them in their API to be the _content_ of the CRL. That's why I added the transformation on our side. But even better now that they do that :D